demos/chips.html

```diff
@@ -55,7 +55,8 @@
 
       <section class="example">
         <h2>Entry Chips</h2>
-        <div class="mdc-chip-set">
+        Add entry: <input id="entry-chip-set-input">
```

Comment:
I think we should add a button here instead of relying on the keyboard event.
Also, the keyboard event doesn't work in IE11. No console errors, but doesn't add a chip.

Reply:
Moved the event listeners to the button instead of input. Also renamed the event handler to simply `addChip`.

Note that this page will be redesigned once entry chips are complete, so the aesthetic of the input is not important at the moment. There's also the possibility of incorporating the input directly within the entry chip set, pending discussion with design.